docs/production-deployment/self-hosted-guide/temporal-nexus.mdx

```diff
@@ -11,6 +11,18 @@ tags:
   - enable-nexus
 ---
 
+import { RelatedReadContainer, RelatedReadItem } from '@site/src/components/related-read/RelatedRead';
+
+:::tip SUPPORT, STABILITY, and DEPENDENCY INFO
+
+**Temporal Nexus** is available in [Public Preview](/evaluate/development-production-features/release-stages#public-preview).
```

Comment:
Please unbold all the Temporal Nexus. We want to aim for consistency across the docs which means every feature that comes out will have to be bold. If every feature is in bold, then it makes little difference.